ethminer/MinerAux.h

```diff
@@ -654,6 +670,12 @@ class MinerCLI
 			<< "    --exit Stops the miner whenever an error is encountered" << endl
 			<< "    -SE, --stratum-email <s> Email address used in eth-proxy (optional)" << endl
 			<< "    --farm-recheck <n>  Leave n ms between checks for changed work (default: 500). When using stratum, use a high value (i.e. 2000) to get more stable hashrate output" << endl
+			<< "    -P URL Specify the URL of a pool endpoint. Can be specified once for the primary pool, then again for the failover pool." << endl
```

Comment:
i would change the wording, that it is supported to use the ``-P`` argument for as many times as you want.
First one specifies the initial pool, all following adds failovers.

Also i mentioned that an official URI supports username+password too. So maybe we need to add this in the example.

``scheme://0xXXXXXXXXXXXXXXX.worker@hostname:port``
or
``scheme://0xXXXXXXXXXXXXXXX.worker:password@hostname:port``